core/src/main/java/cucumber/runtime/table/DataTableDiff.java

```diff
@@ -0,0 +1,39 @@
+package cucumber.runtime.table;
+
+import cucumber.api.DataTable;
+import gherkin.pickles.PickleRow;
+import gherkin.pickles.PickleTable;
+
+import java.util.AbstractMap.SimpleEntry;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataTableDiff extends DataTable {
```

Comment:
This and other classes are all public but don't need to be. They're not part of the public API as far as I can see.